lib/KDateRange/KDateInput.vue

```diff
@@ -28,9 +28,10 @@
 
 <script>
 
-  import { v4 as uuidv4 } from 'uuid';
   import KTextBox from '../KTextbox';
 
+  let uuid = 0;
```

Comment:
If we used the same approach from another component (instead of using `uuid` library), could there be any conflicts?

Reply:
Only if the ID prefix wasn't already unique